src/main/java/Duke.java

```diff
@@ -1,10 +1,94 @@
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Task[] list = new Task[100];
+        int count = 0;
+
+            System.out.println("____________________________________________________________\n  " +
+                    " Hello! I'm Duke\n" +
+                    " What can I do for you?\n" +
+                    "____________________________________________________________\n");
+
+            while(true) {
+                Scanner newScanner = new Scanner(System.in);
+                String user_input = newScanner.nextLine();
+                String[] user_input_split = user_input.split(" ");
+
+                if (user_input_split[0].equals("bye") | user_input_split[0].equals("Bye")) {
+                    System.out.println(" Bye. Hope to see you again soon!\n" +
+                            "____________________________________________________________\n");
+                    break;
+                }
+
+                else if (user_input_split[0].contains("mark")) {
+                    if (user_input_split[0].contains("unmark")) {
```

Comment:
Could maybe use .equalsIgnoreCase() instead of .contains() or .equals() for easier coding